pokeapp/src/components/Sidebar.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const SideBar = () => {
+
+    return(
+        <div className="sidebar">
+            SideBar
+        </div>
+    )
+};
+
+export default SideBar;
```

Comment:
As you add more stuff, this could get bigger, and so you are doing a great job on setting this up